src/views/forms/TransferForm.tsx

```diff
@@ -85,9 +89,13 @@ export const TransferForm = ({
   }, [fee]);
 
   const asset = (token ?? selectedAsset) as DydxChainAsset;
+  const isChainTokenSelected = asset === DydxChainAsset.CHAINTOKEN;
   const isUSDCSelected = asset === DydxChainAsset.USDC;
   const amount = isUSDCSelected ? size?.usdcSize : size?.size;
   const showNotEnoughGasWarning = fee && isUSDCSelected && usdcBalance < fee;
+  const showMemoField = showMemoTransferField && isChainTokenSelected;
+  const showMemoEmptyWarning = showMemoField && !memo;
```

Comment:
is there a way to only show this after address/amount is filled? i think there were previous complaints when we just say address is invalid when no address has been filled

Reply:
oh hm yeah i can def make edits to do that!